client-frameworks-support/client-support-angular/projects/client-support-angular/README.md

```diff
@@ -1,24 +1,115 @@
 # ClientSupportAngular
 
-This library was generated with [Angular CLI](https://github.com/angular/angular-cli) version 11.0.0.
+This library provides several features to run your Angular application inside the Luigi micro frontend framework.  
+If you want to know more about Luigi, please have a look at the [Luigi homepage](https://luigi-project.io/).
 
-## Code scaffolding
+## How to use the library
+1. Import the library in the `package.json`:
+```javascript
+npm install @luigi-project/fundamental-ngx-sample-apps-master -s
+```
 
-Run `ng generate component component-name --project client-support-angular` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module --project client-support-angular`.
-> Note: Don't forget to add `--project client-support-angular` or else it will be added to the default project in your `angular.json` file. 
+2. Once the library is imported and saved in your Angular project, you need to import the module `LuigiAngularSupportModule`:
 
-## Build
+```javascript
+imports: [
+  ........
+  ,LuigiAngularSupportModule
+],
+```
 
-Run `ng build client-support-angular` to build the project. The build artifacts will be stored in the `dist/` directory.
+## Features
+These are the main features provided by the library:
+* [LuigiContextService](#LuigiContextService)
+* [LuigiAutoRoutingService](#LuigiAutoRoutingService) 
 
-## Publishing
+### LuigiContextService
+You can inject this service inside your Angular items in order to:
+* Get the current (latest) Context that we received from Luigi Core
+* Provide an Observable<Context> where through subscribing, you can get any Context change     
+    
+**LuigiContextService** is an abstract class. Its implementation is in the **LuigiContextServiceImpl** class.  
+If you need to change or extend the implementation, you can easily create a new class extending **LuigiContextServiceImpl**:
 
-After building your library with `ng build client-support-angular`, go to the dist folder `cd dist/client-support-angular` and run `npm publish`.
+In this class, we added the possibility to "reuse" a component and not initialize it every time you load it (it could be useful to keep component state.)  
 
-## Running unit tests
+```javascript
+export class YourContextService extends  LuigiContextServiceImpl {
+    ....    
+}
 
-Run `ng test client-support-angular` to execute the unit tests via [Karma](https://karma-runner.github.io).
+```
+Inside your module, redefine the provider:
+ ```javascript
+providers: [
+    {
+        provide: LuigiContextService,
+        useClass: YourContextService
+    }
+]
+ ```
+    
+### LuigiAutoRoutingService
+This service you cannot directly be used, but it will provide useful features how to synchronize your angular application with Luigi navigation.  
```

Comment:
```suggestion
This service cannot be used directly, but it will provide useful features on how to synchronize your angular application with Luigi navigation.  
```
unless @alexandra-simeonova  has a better suggestion